tests/utils/initial-state.json

```diff
@@ -14,5 +14,7 @@
   },
   "fees": {},
   "settings": {},
-  "reserved": {}
+  "reserved": {},
+  "foundation": {},
+  "vaults": {}
```

Comment:
we'll likely want to also update `inital-state.json` in `src` directory

Reply:
No need to do this one, its already updated.